# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 100  

# Position Controllers ---------------------------------------
jleft_rear_wheel_position_controller:
  type: effort_controllers/JointPositionController
  joint: jleft_rear_wheel
  pid: {p: 10.0, i: 0.00, d: 0.00}
jright_rear_wheel_position_controller:
  type: effort_controllers/JointPositionController
  joint: jright_rear_wheel
  pid: {p: 10.0, i: 0.00, d: 0.00}
jleft_front_wheel_position_controller:
  type: effort_controllers/JointPositionController
  joint: jleft_front_wheel
  pid: {p: 10.0, i: 0.00, d: 0.00}
jright_front_wheel_position_controller:
  type: effort_controllers/JointPositionController
  joint: jright_front_wheel
  pid: {p: 10.0, i: 0.00, d: 0.00}